docs/contributing/index.md

```diff
@@ -19,9 +19,9 @@ Like Autoware itself, these guidelines are being actively developed and suggesti
 
 ## Code of Conduct
 
-To ensure the Autoware community stays open and inclusive, please follow the [code of conduct](https://github.com/autowarefoundation/autoware/blob/main/CODE_OF_CONDUCT.md).
+To ensure the Autoware community stays open and inclusive, please follow the [Code of Conduct](https://github.com/autowarefoundation/autoware/blob/main/CODE_OF_CONDUCT.md).
```

Comment:
Seeing this file, `Code of Conduct` is used in most places.
https://github.com/autowarefoundation/autoware/blob/44c70d33825617b56d8de5e6fe921000238238bd/CODE_OF_CONDUCT.md